### PR TITLE
Fix for SBSConflicts?

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

**Comment by jzila** [comment]
Still fails intermittently, haven't fixed it yet...

**Comment by jzila** [comment]
@akalin-keybase ran this 18 times and no SBSConflicts reproductions.

File: libkbfs/tlf_handle.go [hunk 1]
```diff
@@ -138,34 +138,35 @@ func (h TlfHandle) ConflictInfo() *TlfHandleExtension {
...
-// also be nil.) Otherwise, the given conflict info must match the
-// existing one.
-func (h *TlfHandle) UpdateConflictInfo(
-	codec Codec, info *TlfHandleExtension) error {
-	if h.conflictInfo == nil {
+// WithUpdatedConflictInfo returns a new handle with the conflict info set to
+// the given one, if the existing one is nil. (In this case, the given one may
+// also be nil.) Otherwise, the given conflict info must match the existing
+// one.
+func (h TlfHandle) WithUpdatedConflictInfo(
```
Comment (akalin-keybase) [inline libkbfs/tlf_handle.go]: Can you add a todo to fix `SetFinalizedInfo`, too?

**Comment by akalin-keybase** [comment]
This lgtm after comment!

**Comment by jzila** [comment]
Test failure is known (different) flake.

**Pull request merged**